src/software/thunderscope/thunderscope_main.py

```diff
@@ -82,7 +82,6 @@
         action="store",
         help="Replay folder for the blue full_system",
         default=None,
-        type=os.path.abspath,
```

Comment:
Curious why this was removed? In case we're bringing it back, I noticed that it also should be added to the `--yellow_log` arg.